docs/basics/101-138-sharethirdparty.rst

```diff
@@ -0,0 +1,502 @@
+   ``experiment.txt`` is fine for everyone to see, ``subject_1.dat`` and
+   ``subject_2.dat`` contain personal and potentially identifying data that
+   can not be shared. Nevertheless, you want collaborators to know that these
+   files exist. The use case
+
+   .. todo::
+
+      Write use case "external researcher without data access"
+
+   details such a scenario and demonstrates how external collaborators (with whom data
+   can not be shared) can develop scripts against the directory structure and
+   file names of a dataset, submit those scripts to the data owners, and thus still perform an
+   analysis despite not having access to the data.
+
+   By publishing only the file contents of ``experiment.txt`` with
+
+   .. code-block:: bash
+
+      $ datalad publish --to github --transfer-data auto experiment.txt
+
+   only meta data about file availability of ``subject_1.dat`` and ``subject_2.dat``
+   exists, but as these files' annexed data is not published, a :command:`datalad get`
+   will fail. Note, though, that :command:`publish` will publish the complete
+   dataset history (unless you specify a commit range with the ``--since`` option
+   -- see the `manual <http://docs.datalad.org/en/latest/generated/man/datalad-publish.html>`_
+   for more information).
+
+
+From the perspective of whom you share your dataset with...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your friend would now want to get your dataset including the annexed
+contents, and you made sure that they can access the Dropbox folder with
+the annexed files (e.g., by sharing an access link), here is what they would
+have to do:
+
+If the repository is on GitHub, a :command:`datalad clone` with the URL
+will install the dataset::
+
+   $ datalad clone https://github.com/adswa/DataLad-101.git
+   [INFO   ] Cloning https://github.com/adswa/DataLad-101.git [1 other candidates] into '/Users/awagner/Documents/DataLad-101'
+   [INFO   ]   Remote origin not usable by git-annex; setting annex-ignore
+   [INFO   ] access to 1 dataset sibling dropbox-remote not auto-enabled, enable with:
+   |         datalad siblings -d "/Users/awagner/Documents/DataLad-101" enable -s dropbox-remote
+   install(ok): /Users/awagner/Documents/DataLad-101 (dataset)
+
+Pay attention to one crucial information in this output::
+
+   [INFO   ] access to 1 dataset sibling dropbox-remote not auto-enabled, enable with:
+   |         datalad siblings -d "/Users/awagner/Documents/DataLad-101" enable -s dropbox-remote
+
+This means that someone who wants to access the data from dropbox needs to
+enable the special remote.
+For this,  this person first needs to install and configure ``rclone``
+as well: Since ``rclone`` is the protocol with which
+annexed data can be transferred from and to Dropbox, anyone who needs annexed
+data from Dropbox needs this special remote. Therefore, the first steps are
+the same as before:
+
```

Comment:
And because they are identical, I would not repeat them verbosely. Just like `install rclone as described before` etc.